data/tooltips.txt

```diff
@@ -482,5 +482,8 @@ tip "limited turn?"
 tip "solar power?"
 	`This ship is powered entirely by solar energy. If you fly too far from the center of a star system, maneuvering may become difficult because your engines will only have a fraction of the power they require.`
 
+tip "fuel?"
+	`This ship has not enough fuel to generate energy.`
```

Comment:
The expression `<verb> not` is seldom seen in modern grammar; `does not <verb>` is much more common. So `This ship does not have enough fuel` would be better.
Also, this tip should have an introduction like `"solar power?"`, such as `This ship relies entirely on its fuel for energy.` It really feels out of place without one.

Reply:
Actually this "fuel?" warning checks only if fuel capacity + generation - consumption > 0, i.e. if it has enough fuel to depart. Anyway, I've rephrased it.